### PR TITLE
adds recent CE glider ops and updates G3 reference designators

### PR DESCRIPTION
**Pull request by s-pearce**
@cwingard can you review my additions of glider deployments 871-00002, 311-00012, and 383-00011 and recoveries 326-00009 and 386-00013?

@rtravis-whoi I included you for review because I updated our G3 to have the new style of reference designator including for a deployment that occurred before the style change (871-00001).  Since you recently updated ref des's for your G3's I thought it would be good to have you review.  Feel free to pass this review on to whoever edited the WHOI G3's if it was not you.
Thanks, 
Stuart

### CODE REVIEW

**Comment by s-pearce** [comment]
Also, I passed the nosetests even after the vocab changes and also added our newest G3 917 into the vocab, though it has not yet been deployed.

File: deployment/CE05MOAS-GL871_Deploy.csv [hunk 1]
```diff
@@ -1,7 +1,13 @@
...
+EK20210120,,SKQ202104S,S.Pearce,CE05MOAS-G0871-00-ENG000000,1,1,2021-01-21T18:05:00,2021-03-24T22:10:00,CGVCE-05MOAS-00000,CGVEH-GLDRCE-00871,CGCON-GLDRCE-00871,44.632,-124.305,,1000,1000,,,
+EK20210120,,SKQ202104S,S.Pearce,CE05MOAS-G0871-01-PARADM000,1,1,2021-01-21T18:05:00,2021-03-24T22:10:00,CGVCE-05MOAS-00000,CGVEH-GLDRCE-00871,CGINS-PARADM-50319,44.632,-124.305,,1000,1000,,,
+EK20210120,,SKQ202104S,S.Pearce,CE05MOAS-G0871-02-FLORTM000,1,1,2021-01-21T18:05:00,2021-03-24T22:10:00,CGVCE-05MOAS-00000,CGVEH-GLDRCE-00871,CGINS-FLORTM-06168,44.632,-124.305,,1000,1000,,,
+EK20210120,,SKQ202104S,S.Pearce,CE05MOAS-G0871-03-ADCPAM000,1,1,2021-01-21T18:05:00,2021-03-24T22:10:00,CGVCE-05MOAS-00000,CGVEH-GLDRCE-00871,CGINS-ADCPAM-73107,44.632,-124.305,,1000,1000,,,
+EK20210120,,SKQ202104S,S.Pearce,CE05MOAS-G0871-04-DOSTAM000,1,1,2021-01-21T18:05:00,2021-03-24T22:10:00,CGVCE-05MOAS-00000,CGVEH-GLDRCE-00871,CGINS-DOSTAM-00864,44.632,-124.305,,1000,1000,,,
+EK20210120,,SKQ202104S,S.Pearce,CE05MOAS-G0871-05-CTDGVM000,1,1,2021-01-21T18:05:00,2021-03-24T22:10:00,CGVCE-05MOAS-00000,CGVEH-GLDRCE-00871,CGINS-CTDGVM-09658,44.632,-124.305,,1000,1000,,,
```
Comment (rtravis-whoi) [inline deployment/CE05MOAS-GL871_Deploy.csv:7]: update to deployment 00001 G3 RefDes looks good

File: vocab/vocab.csv [hunk 2]
```diff
@@ -269,13 +269,20 @@ CE05MOAS-GL386-02-FLORTM000,Coastal Endurance,Mobile Assets,Coastal Glider 386,3
...
-CE05MOAS-GL871-04-DOSTAM000,Coastal Endurance,Mobile Assets,Coastal Glider 871,Dissolved Oxygen,Aanderaa,Optode 4831,0,1000
-CE05MOAS-GL871-05-CTDGVM000,Coastal Endurance,Mobile Assets,Coastal Glider 871,CTD,Sea-Bird,SBE Glider Payload CTD (GP-CTD),0,1000
+CE05MOAS-G0871,Coastal Endurance,Mobile Assets,Coastal Glider 871,,,,0,1000
+CE05MOAS-G0871-00-ENG000000,Coastal Endurance,Mobile Assets,Coastal Glider 871,Mobile Asset Controller,Teledyne Webb,G3 Slocum Gliders,0,1000
+CE05MOAS-G0871-01-PARADM000,Coastal Endurance,Mobile Assets,Coastal Glider 871,Photosynthetically Available Radiation,Biospherical Instruments,QSP-2155,0,1000
+CE05MOAS-G0871-02-FLORTM000,Coastal Endurance,Mobile Assets,Coastal Glider 871,3-Wavelength Fluorometer,WET Labs,ECO Puck FLBBCD-SLC,0,1000
+CE05MOAS-G0871-03-ADCPAM000,Coastal Endurance,Mobile Assets,Coastal Glider 871,Velocity Profiler (600kHz),Teledyne RDI,Explorer DVL 600 kHz,0,1000
+CE05MOAS-G0871-04-DOSTAM000,Coastal Endurance,Mobile Assets,Coastal Glider 871,Dissolved Oxygen,Aanderaa,Optode 4831,0,1000
+CE05MOAS-G0871-05-CTDGVM000,Coastal Endurance,Mobile Assets,Coastal Glider 871,CTD,Sea-Bird,SBE Glider Payload CTD (GP-CTD),0,1000
```
Comment (rtravis-whoi) [inline vocab/vocab.csv:278]: G3 RefDes updates look correct to me

File: vocab/vocab.csv [hunk 3]
```diff
@@ -269,13 +269,20 @@ CE05MOAS-GL386-02-FLORTM000,Coastal Endurance,Mobile Assets,Coastal Glider 386,3
...
+CE05MOAS-G0917,Coastal Endurance,Mobile Assets,Coastal Glider 917,,,,0,1000
+CE05MOAS-G0917-00-ENG000000,Coastal Endurance,Mobile Assets,Coastal Glider 917,Mobile Asset Controller,Teledyne Webb,G3 Slocum Gliders,0,1000
+CE05MOAS-G0917-01-PARADM000,Coastal Endurance,Mobile Assets,Coastal Glider 917,Photosynthetically Available Radiation,Biospherical Instruments,QSP-2155,0,1000
+CE05MOAS-G0917-02-FLORTM000,Coastal Endurance,Mobile Assets,Coastal Glider 917,3-Wavelength Fluorometer,WET Labs,ECO Puck FLBBCD-SLC,0,1000
+CE05MOAS-G0917-03-ADCPAM000,Coastal Endurance,Mobile Assets,Coastal Glider 917,Velocity Profiler (600kHz),Teledyne RDI,Explorer DVL 600 kHz,0,1000
+CE05MOAS-G0917-04-DOSTAM000,Coastal Endurance,Mobile Assets,Coastal Glider 917,Dissolved Oxygen,Aanderaa,Optode 4831,0,1000
+CE05MOAS-G0917-05-CTDGVM000,Coastal Endurance,Mobile Assets,Coastal Glider 917,CTD,Sea-Bird,SBE Glider Payload CTD (GP-CTD),0,1000
```
Comment (rtravis-whoi) [inline vocab/vocab.csv:285]: New G3 glider RefDes look good to me.

**Review by rtravis-whoi — APPROVED** [review]
The G3 glider RefDes updates look good to me.  I added specific 'looks good' comments to the portions I reviewed.

**Review by cwingard — CHANGES_REQUESTED** [review]
The deployment/CE05MOAS-GL871_Deploy.csv file should be renamed to deployment/CE05MOAS-G0871_Deploy.csv

**Comment by s-pearce** [comment]
@cwingard Good catch.  I've rename the file and pushed it to the pull request.

**Review by cwingard — APPROVED** [review]

**Pull request merged**